sass/_FacetFooter.scss

```diff
@@ -10,7 +10,33 @@
   &:hover,
   &:focus {
     background: $color-light-grey;
-    outline: none;
+    &,
+    &.coveo-accessible-button-pressed {
+      outline: 1px solid transparent;
+      /* Works on Firefox, doesn't work on IE */
+      outline-offset: -3px;
+    }
+    /* Works on IE and Edge, doesn't work on Firefox & Chrome */
+    @media screen and (-ms-high-contrast: white-on-black) {
+      &,
+      &.coveo-accessible-button-pressed {
+        background-color: white;
+        .coveo-facet-more-icon-svg,
+        .coveo-facet-less-icon-svg {
+          color: black;
+        }
+      }
+    }
+    @media screen and (-ms-high-contrast: black-on-white) {
+      &,
```

Comment:
What is this `&,` for? Also, why is `@media screen` needed?

Reply:
The `&` is here because I want the style to always be applied, regardless of whether or not the button is pressed. I redundantly apply the same style when the button is pressed since it was being overwritten by a higher-priority rule.

As for the `@media screen`, I copied it from [Mozilla's example](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/-ms-high-contrast#Examples).